modules/scaleio/queries.go

```diff
@@ -62,7 +62,6 @@ var (
         "numOfMappedToAllVolumes",
         "numOfProtectionDomains",
         "numOfRfcacheDevices",
-        "numOfScsiInitiators",
```

Comment:
i think better to comment out it and add a comment with a reason

Reply:
Good point. Modified